app/views/content/life-as-a-teacher/change-careers/how-to-change-careers-to-become-a-teacher/_article.html.erb

```diff
@@ -0,0 +1,43 @@
+    <h2 class="heading--box-blue">What you'll need to become a teacher</h2>
+    <p>You’ll need a degree and <a href="/train-to-be-a-teacher/what-is-qts">qualified teacher status (QTS)</a> to work in the majority of schools in England. This includes state maintained primary, secondary and special schools.</p>
+    <p>There are different ways to get QTS.</p>
+    <h3>If you have a degree</h3>
+    <p>If you already have a degree, you can do a postgraduate teacher training course to gain QTS.</p>
+    <p>It does not matter when you graduated, and your degree subject does not necessarily need to be in the subject you want to teach. Once you have QTS, you can also teach any other subject, as long as you can show you have enough expertise.</p>
+    <h4>Funding postgraduate teacher training</h4>
+    <p>Most postgraduate teacher training courses charge fees. There are ways to fund your training, for example, through tuition fee and maintenance loans. If you train to teach certain subjects, you may also be eligible for a bursary or scholarship of up to <%= v :scholarships_generic_maxshortened %>.</p>
+    <p>Other postgraduate teacher training courses come with a salary. This means you will not pay tuition fees and will receive a salary while you train. These courses are in high demand and very competitive, so it’s important to apply as soon as you can if you’re eligible.</p>
+    <p>If you’re eligible for a bursary or scholarship, you could receive more money on a non-salaried course than on a salaried one.</p>
+    <p><a href="/train-to-be-a-teacher/if-you-have-a-degree">Find out more about postgraduate teacher training</a>.</p>
+    <h3>If you do not have a degree</h3>
+    <p>If you do not have a degree, you can:</p>
+    <ul>
+      <li>do undergraduate teacher training to get a degree alongside qualified teacher status (QTS)</li>
+      <li>check if you're eligible to do a teaching degree apprenticeship</li>
+    </ul>
+    <h4>Funding undergraduate teacher training </h4>
+    <p>Undergraduate teacher training courses can be funded through tuition fee and maintenance loans. You may also be eligible for a bursary of <%= v :bursaries_undergraduate %>.</p>
+    <p>Teaching degree apprenticeships allow you work in a school and earn a salary while getting a bachelor’s degree, so you do not have to pay tuition fees.</p>
+    <p><a href="/train-to-be-a-teacher/if-you-dont-have-a-degree">Find out more about training to teach if you do not have a degree</a>.</p>
+    <h3>If you do not want to get a degree</h3>
+    <p>You do not need a degree and QTS to <a href="/life-as-a-teacher/age-groups-and-specialisms/further-education-teachers">teach in further education</a> or to <a href="/life-as-a-teacher/age-groups-and-specialisms/early-years-teachers">teach in early years</a>. So if you want to become a teacher, but are not able to get a degree, you can explore these options.</p>
+  <h2 class="heading--box-blue">Find out more</h2>
```

Comment:
I think this h2 needs more clarity - for screen reader users, it has no context of what you are finding out more about?